src/dmd/globals.h

```diff
@@ -123,7 +123,8 @@ struct Param
     bool fix16997;      // fix integral promotions for unary + - ~ operators
                         // https://issues.dlang.org/show_bug.cgi?id=16997
     bool vsafe;         // use enhanced @safe checking
-    bool ehnogc;        // use @nogc exception handling
+    bool cpp98;             // follow C++98 type system issues rather than C++11
```

Comment:
`type system issues`
 type mangling rules?

Reply:
It's a little more than just mangling.